components/proxy/src/test/java/com/hotels/styx/serviceproviders/ServiceProvisionTest.java

```diff
@@ -131,7 +223,61 @@ public void servicesReturnEmptyWhenFactoryKeyDoesNotExist() {
 
     @Test(expectedExceptions = Exception.class, expectedExceptionsMessageRegExp = "(?s).*No such class 'my.FakeClass'.*")
     public void throwsExceptionWhenClassDoesNotExist() {
-        loadService(environment.configuration(), environment, "not.real", String.class);
+        loadRetryPolicy(environment.configuration(), environment, "not.real", String.class);
+    }
+
+    @Test(expectedExceptions = ConfigurationException.class,
+            expectedExceptionsMessageRegExp = "Unexpected configuration object 'services.factories.backendProvider', Configuration.*'")
+    public void informsAboutServiceFactoryConfig() {
```

Comment:
These `inform...` test names are not very informative.

Reply:
Yeah, I agree. I will address this.